src/api/record/record.ts

```diff
@@ -297,4 +308,17 @@ export class RecordHandler extends RupertPlugin {
     });
     return Promise.all(images.map(deleteImage));
   }
+
+  private removeData(record: Record): Promise<void> {
+    return new Promise<void>((s, j) => {
+      unlink(this.getRecordDBPath(record), (err: any) => {
+        if (err !== null) { return j(err); }
+        s();
+      });
+    });
+  }
+
+  private getRecordDBPath(record: Record): string {
+    return join(this.dataPath, record.id, 'db.json');
```

Comment:
Instead of db.json, can we have the name the same as the tape label? So it should always be the same as the folder, `rita_1/rita_1.json`

I realize this means we have to write logic for renaming, too.